source/extensions/stat_sinks/open_telemetry/open_telemetry_impl.cc

```diff
@@ -33,7 +34,8 @@ OpenTelemetryGrpcMetricsExporterImpl::OpenTelemetryGrpcMetricsExporterImpl(
     const OtlpOptionsSharedPtr config, Grpc::RawAsyncClientSharedPtr raw_async_client)
     : config_(config), client_(raw_async_client),
       service_method_(*Protobuf::DescriptorPool::generated_pool()->FindMethodByName(
-          "opentelemetry.proto.collector.metrics.v1.MetricsService.Export")) {}
+          std::string(Envoy::Extensions::Common::OpenTelemetry::Sdk::Metrics::Constants::
+                          METRICS_SERVICE_EXPORT_METHOD))) {}
```

Comment:
You can drop `Envoy::Extensions::` I think, since the final string is longer than the original :)

Reply:
Also, are you sure you need `std::string(`? I think gRPC takes string_views these days.